src/element-stub.js

```diff
@@ -15,14 +15,31 @@
  */
 
 import {BaseElement} from './base-element';
+import {getMode} from '../src/mode'
 
 
 /** @type {!Array} */
 export const stubbedElements = [];
 
+let ampAdScriptInserted = false;
+
 export class ElementStub extends BaseElement {
   constructor(element) {
     super(element);
+    if((element.tagName.toLowerCase() == "amp-ad"
+      || element.tagName.toLowerCase() == "amp-embed") 
+      && (!ampAdScriptInserted)) {
+      var ampAdScript = this.getWin().document.createElement('script');
+      ampAdScript.async = true;
+      ampAdScript.setAttribute('custom-element', 'amp-ad');
+      if (getMode().localDev) {
+        ampAdScript.src = "http://localhost:8000/dist/v0/amp-ad-0.1.max.js";
+      } else {
+        ampAdScript.src = "https://cdn.ampproject.org/v0/amp-ad-0.1.js";
```

Comment:
@zhouyx once you rebase, you will now be able to use `getMode().version`, then do something like:

```js
let domain = getMode().localDev ? 'http://localhost:8000/' : 'https://cdn.ampproject.org/';
let folderPath = getMode().version == "$internalRuntimeVersion" ? '' : `rtv/${getMode().version}/`;
let scriptSrc = `${domain}${folderPath}v0/amp-ad-0.1.js`;
```